content/news/release-20150425.md

```diff
@@ -0,0 +1,61 @@
+We are very pleased to announce the Spring release of OpenRA.  This is a huge step forward for the Dune 2000 mod in particular, but also, behind the scenes, for the still unreleased, work-in-progress Tiberian Sun mod which has received support for some crucial features during this cycle.
```

Comment:
Actually, this is repeated below, so I suggest removing it, and then combining the first sentence with the second paragraph.